gtest/src/program.rs

```diff
@@ -917,8 +917,23 @@ mod tests {
         assert_eq!(prog.balance(), (2 + 4 + 6) * crate::EXISTENTIAL_DEPOSIT);
 
         // Request to smash the piggy bank and send the value to the receiver address
-        prog.send_bytes(receiver, b"smash");
-        sys.claim_value_from_mailbox(receiver);
+        let res = prog.send_bytes(receiver, b"smash");
+        let reply_to_id = {
+            let log = res.log();
+            // 1 auto reply and 1 message from program
+            assert_eq!(log.len(), 2);
+
+            let core_log = log
+                .iter()
+                .find(|&core_log| {
+                    core_log.eq(&Log::builder().dest(receiver).payload_bytes(b"send"))
+                })
+                .expect("message not found");
+
+            core_log.id()
+        };
+
+        assert!(sys.claim_value_from_mailbox(receiver, reply_to_id).is_ok());
```

Comment:
I'd leave it as 1) it was initially in the API and 2) there are no strong reasons to remove it.

Reply:
> 1) it was initially in the API 

We are allowed to break the API

> 2) there are no strong reasons to remove it.

To avoid duplication of functionality and make it consistent

Reply:
done